helpers/plugins.js

```diff
@@ -0,0 +1,26 @@
+const pluginSets = new Map();
+
+export function getPlugin(setKey, pluginKey) {
+	const pluginSet = pluginSets.get(setKey);
+	return pluginSet?.plugins.find(plugin => plugin.options.key === pluginKey)?.plugin;
+}
+
+export function getPlugins(setKey) {
+	const pluginSet = pluginSets.get(setKey);
+	if (!pluginSet) return [];
```

Comment:
Again, wondering if this should just `throw`, or do you think the possibility of a plugin set having no implementations is plausible?

Reply:
I could go either way on this, but my current feeling is that it's the consumer's decision whether no plugins is an error. Getting no plugins back in my case would certainly be unexpected. But maybe there's a scenario where some behaviours or features are enabled through user configuration, which in turn controls whether certain plugins are registered. In such case, maybe we'd expect no plugins. 🤷‍♂️